WORKSPACE

```diff
@@ -1,5 +1,16 @@
 workspace(name = "envoy_mobile")
 
+# TODO[fz]: Remove once envoy updates PGV
```

Comment:
Specifically, any release including this PR: https://github.com/envoyproxy/protoc-gen-validate/pull/533